ChaosMod/Effects/db/Player/PlayerLaunchUp.cpp

```diff
@@ -18,13 +18,13 @@ static void OnStart()
 		SET_PED_TO_RAGDOLL(playerPed, 10000, 10000, 0, true, true, false);
 		WAIT(0);
 
-		SET_ENTITY_VELOCITY(playerPed, playerVel.x, playerVel.y, 100.f);
+		SET_ENTITY_VELOCITY(playerPed, playerVel.x, playerVel.y, 300.f);
```

Comment:
Too much IMO, it’s like „teleport to heaven“ LUL

Reply:
Balance is a big no-no for my version.